packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -131,14 +131,13 @@ class Send extends Component {
                     }) => (
                       <form className='send-form' onSubmit={handleSubmit}>
                         <fieldset className='form_fields'>
-                          <Field
+                          <FetherForm.Field
```

Comment:
Revert. This makes the whole TxForm broken. Field is a render props wrapper around FetherForm.Field, which uses final-form.